src/common/components/loading/loading.scss

```diff
@@ -0,0 +1,3 @@
+.loading-wrap {
+  transform: translate(-50%, -50%);
```

Comment:
@Omicron7 This should fix the loading indicators not being aligned on their centers. @riceguitar suggested it. I removed your `position: static;` in the loadingOverlay.scss. I'm not sure how much of the alignment stuff in your `loading-background` is still needed.